dmc_configs/dmc_configs/README.md

```diff
@@ -0,0 +1,176 @@
+- Voting over 1, 2, 4, 8, or 16 LMs
+
+The main output measure is accuracy and rotation error as a function of number of LMs.
+
+**TODO:**
+- Config builders for arbitrary numbers of LMs are not currently included in `dmc_eval_experiments.py`.
+- Comparable configs should be used to generate views for evaluated rotations to pass to ViT model for comparison.
+
+## Figure 5: Rapid Inference with Model-Based Policies
+
+Consists of 3 experiments:
+- `dist_agent_1lm_randrot_noise_nohyp` - No hypothesis-testing
+- `dist_agent_1lm_randrot_noise_moderatehyp` - Occasional hypothesis-testing
+  - Should specify:
+    - elapsed_steps_factor=20
+    - min_post_goal_success_steps=10
+- `dist_agent_1lm_randrot_noise` - Default, frequent hypothesis-testing
+  - Should specify:
+    - elapsed_steps_factor=10
+    - min_post_goal_success_steps=5
+
+This means performance is evaluated with:
+- 77 objects
+- Sensor noise and 14 random rotations
+- No voting
+- Varying levels of hypothesis-testing
+
+The main output measure is accuracy and rotation error as a function of hypothesis-testing policy.
+
+**TODO:**
+- These configs need to be specified.
+
+## Figure 6: Rapid Learning
+
+Consists of 6 experiments:
+- `dist_agent_1lm_randrot_nohyp_1rot_trained`
+- `dist_agent_1lm_randrot_nohyp_2rot_trained`
+- `dist_agent_1lm_randrot_nohyp_4rot_trained`
+- `dist_agent_1lm_randrot_nohyp_8rot_trained`
+- `dist_agent_1lm_randrot_nohyp_16rot_trained`
+- `dist_agent_1lm_randrot_nohyp_32rot_trained`
+
+This means performance is evaluated with:
+- 77 objects
+- 14 random rotations
+- NO sensor noise*
+- NO hypothesis-testing*
+- No voting
+- Varying numbers of rotations trained on (evaluations use different baseline models)
+
+*No hypothesis-testing as the ViT model comparison only receives one view and cannot move around object, and no noise since Sensor-Module noise does not have a clear analogue for the ViT model.
+
+The main output measure is accuracy and rotation error as a function of training rotations.
+
+**Notes:**
+- Training rotations should be structured:
+  1. First 6 rotations = cube faces
+  2. Next 8 rotations = cube corners
+  3. Remaining = random rotations (as otherwise introduces redundancy)
+
+**TODO:**
+- Configs need to be specified
+- Comparable configs needed for ViT model comparison
+
+## Figure 7: Computationally Efficient Learning and Inference
+
+Consists of 8 experiments:
+
+### Inference (7 experiments):
+- `dist_agent_1lm_randrot_nohyp_x_percent_5p` - 5% threshold
+- `dist_agent_1lm_randrot_nohyp_x_percent_10p` - 10% threshold
+- `dist_agent_1lm_randrot_nohyp_x_percent_20p` - 20% threshold (default for other experiments)
+- `dist_agent_1lm_randrot_nohyp_x_percent_40p` - 40% threshold
+- `dist_agent_1lm_randrot_nohyp_x_percent_80p` - 80% threshold
+- `dist_agent_1lm_randrot_nohyp_x_percent_100p` - 100% threshold
+
+### Training (1 experiment):
+- `dist_agent_77obj_1rot_trained`*
+
+This means performance is evaluated with:
```

Comment:
Why would we evaluate performance here? Isn't this just about FLOPs during training?

Reply:
Yeah this was meant to refer to the inference side of things (where we also want to look at accuracy), I've re-ordered it to be clearer.